pyinteraph/path_analysis.py

```diff
@@ -350,15 +367,27 @@ def plot_graph(fname, graph, hub_num, col_map_e, col_map_n, dpi):
         nx.draw_networkx_nodes(graph, 
                                pos, 
                                nodelist = list(hubs),
-                               node_size = 900,
+                               node_size = node_size,
                                node_color = 'gray',
                                edgecolors = 'black')
-    elif unique_hubs > 1:
+        # Add label for single unique hub (temporary, fix later)
+        label = matplotlib.patches.Patch(color='gray', label=np.unique(hubs_deg)[0])
```

Comment:
You can make the colour of the label a parameter of the function as well, so that it is easier to modify it.